packages/zero-cache/src/db/lite-tables.ts

```diff
@@ -60,12 +63,19 @@ export function listTables(db: Database): LiteTableSpec[] {
       tables.push(table);
     }
 
+    const elemPgTypeClass = isArray(col.type)
+      ? isEnum(col.type)
+        ? PostgresTypeClass.Enum
+        : PostgresTypeClass.Base
```

Comment:
Base is the super type of all type in PG or something?

Reply:
This is not entirely correct. I should add a comment... there are more types and this fells a bit unclean